ct-aa/dotTheory---JavaScript
# ✨ JS-610: Create a JSON File

# Related Tickets
- [JS-610](https://codeandtheory.atlassian.net/browse/JS-610)

# Description
Added the **Create a JSON File** challenge solutions.

# Validation
* Open the **Create a JSON File** folder.
* Open the songs.json file.
* The file should contain an array with one object.

# Risks
N/A
